index.html

```diff
@@ -78,79 +78,7 @@ <h1 class="headline__title">I’m Ahmed H. Glad to see you!</h1>
   </section>
 
   <section class="works" id="portfolio">
```

Comment:
- [ ] Please also make sure your design follows the Figma design. I would recommend making them in a `Z` pattern, however, you have used the `E` pattern. You may see the difference from the screenshots down below. 🙂📸

Your design

<img width="1042" alt="Screen Shot 2022-01-26 at 10 17 37 PM" src="https://user-images.githubusercontent.com/69011963/151214550-977ffb25-7c7f-4bdb-a83b-2f7dd639c765.png">


Figma design

![image](https://user-images.githubusercontent.com/69011963/151214421-9a7fd72c-a59f-4f6d-bf13-75c18a7f0ce7.png)

Reply:
Addressed 😎